inst/WORDLIST

```diff
@@ -2,6 +2,7 @@ Biomarker
 CDISC
 Forkers
 Hoffmann
+Javascript
```

Comment:
If we put `JavaScript` in quotes in the documentation, then you will not have this entry

Reply:
`JavaScript` is a known well spelled word. It was removed from this file with your previous suggestion :+1: